src/adapter/src/coord/message_handler.rs

```diff
@@ -206,6 +206,14 @@ impl Coordinator {
                         )
                         .await;
                 }
+                Message::CheckSchedulingPolicies(span) => {
+                    // Add an OpenTelemetry link to our current span.
+                    tracing::Span::current().add_link(span.context().span().span_context().clone());
```

Comment:
Because `check_scheduling_policies` is annotated with `#[mz_ore::instrument(level = "debug")]` I don't think you need to manually do any instrumenting here

Reply:
Ok, I removed it.